AsyncDisplayKit/Layout/ASBackgroundLayoutSpec.mm

```diff
@@ -23,20 +23,21 @@ @interface ASBackgroundLayoutSpec ()
 
 @implementation ASBackgroundLayoutSpec
 
-+ (instancetype)newWithChild:(id<ASLayoutable>)child background:(id<ASLayoutable>)background
+- (instancetype)initWithChild:(id<ASLayoutable>)child background:(id<ASLayoutable>)background
 {
-  if (child == nil) {
-    return nil;
+  self = [super init];
```

Comment:
It's an atypical style, but ASDK uses:

if (!(self = [super init])) {
  return nil;
}

Thus, the remainder of the method is not indented.